### DISCUSSION TITLE
Actions and Packages - Check in  🆕

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#36130] [Actions]
Hello everyone and hope you’re having a great October so far. We’re back with the latest from the Actions and Packages world. 🌎 

We'd love to hear your feedback on [GitHub Larger Hosted Runners Beta](https://github.blog/changelog/2022-09-01-github-actions-larger-runners-are-now-in-public-beta/). Please fill out [this](https://github.surveymonkey.com/r/P2YNVND) short, anonymous survey to help my team understand your experience & opportunities for improvement. [@lkfortuna](https://github.com/lkfortuna) is leading the efforts to take your feedback to the future product ships. For more information, please visit [this](https://github.com/orgs/community/discussions/35265) discussion. 💯 

### [Introducing Actions on GitHub Mobile 📱 ](https://github.blog/changelog/2022-10-04-introducing-actions-on-github-mobile/) 😲 

Yes, you heard it right! Actions are coming to GitHub Mobile! From a tap 🤌   to viewing and reviewing pull requests to defining a workflow-run on the go, 🚶we’ve got you covered! [Read more about GitHub Mobile](https://github.com/mobile) and [send us your feedback](https://github.com/orgs/community/discussions/categories/mobile) to help us improve. Got more feedback to share? ☑️  You can contribute to several GitHub projects including GitHub Docs by participating in this year's [Hacktoberfest](https://github.blog/2022-10-04-expand-your-open-source-contributions-during-hacktoberfest-2022/). If you’ve ever wanted to contribute to GitHub’s open source documentation, now is your chance! ✌️ 

### From the Community - Citizens on a Roll! 🎢 

- Multiple posts by [@antonblr](https://github.com/antonblr) had me read 👀  almost all of their contributions in the community and here are a few examples of their workmanship.
  - Here’s a recent thread in which [@antonblr](https://github.com/antonblr) provided a great solution to [@scarf005's](https://github.com/scarf005) question Well Done! 👏 
  - Solved ✅  [another one](https://github.com/orgs/community/discussions/35565) in collaboration with a great suggestion by [@ChristopherHX](https://github.com/ChristopherHX) 🤝 
  - This one is a masterpiece🥇 with links to all the docs that [@dpo](https://github.com/dpo) would ever need fort workflows.  
  - We’re not done with [@antonblr's](https://github.com/antonblr) contributions yet, [another gem 💎  ](https://github.com/community/community/discussions/35109) with a one liner solution. This is great!  

- [@ChristopherHX](https://github.com/ChristopherHX) has been providing some amazingly descriptive ✍️  answers in the community.
  - [@kuisathaverat](https://github.com/kuisathaverat) has been looking to [define an environment variable](https://github.com/orgs/community/discussions/35739) Global to a job but did not use the correct environment which [@ChristopherHX](https://github.com/ChristopherHX) did in his really well crafted solution. 💯 
  - [Sharing](https://github.com/community/community/discussions/29796) the updated info ℹ️  with the author of [actionlint](https://github.com/rhysd/actionlint). [@rhysd](https://github.com/rhysd) was not sure about an indented behavior of reusable workflow. [@ChristopherHX](https://github.com/ChristopherHX) helped with a detailed response and his documentation is spot on! 🏅   
- Let’s look at this [self-solve ✔️  ](https://github.com/orgs/community/discussions/35827) by [@schalkneethling](https://github.com/schalkneethling) realizing that the default GITHUB_TOKEN would’ve solved the error. Thanks for coming back and updating the discussion. ⚡ 

### GitHub Universe - Agenda is Live! 🌐 

We’re super excited 😮  for GitHub Universe and the countdown 📅   has begun. It is less than a month from now and we have the list of agenda on our hands. You will have the chance to explore 80+ content sessions delivered by over 120 different speakers, across two days and four content tracks, all designed to level up your skills. Read more about the agenda [here](https://github.blog/2022-10-11-the-github-universe-2022-agenda-is-live/). To get even more excited check out the [two great ways](https://github.blog/2022-10-05-two-ways-you-can-experience-github-universe/) to experience everything GitHub Universe has to offer this year from [@lclindeman](https://github.com/lclindeman)  🚀  

#### GitHub Actions: [Deprecating save-state and set-output commands](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) 🆕 

FYI, to avoid untrusted logged data to use set-state and set-output workflow commands without the intention of the workflow author we have introduced a new set of environment files to manage state and output. [Here’s more details](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) on the change.

#### GitHub Actions: [GitHub Actions: larger Hosted Runners now support `-latest` tag](https://github.blog/changelog/2022-10-12-github-actions-larger-hosted-runners-now-support-latest-tag/) 🏷️ 

GitHub now supports using the -latest image label on your larger hosted runners. With the -latest label developers can be sure that their workflows use the latest available GA image in GitHub Actions. Are you interested in learning more about larger hosted runners? Read the announcement [here](https://github.blog/changelog/2022-09-01-github-actions-larger-runners-are-now-in-public-beta/) or sign up for the beta [here](https://github.com/features/github-hosted-runners/signup). 📢

#### GitHub Actions: [Jobs running on `macos-latest` are now running on macOS 12](https://github.blog/changelog/2022-10-03-github-actions-jobs-running-on-macos-latest-are-now-running-on-macos-12/) 🖱️ 

This is a much anticipated change, and over the next 10 weeks, jobs using the macos-latest runner label will migrate from macOS 11 to 12. You just need to include runs-on: macos-latest in your jobs.

#### Packages: [Consolidated owner and admin role](https://github.blog/changelog/2022-10-10-packages-consolidated-owner-and-admin-role/) 📦 

We’re merging identical privileges that the admin role and original poster of the package used to have into a single authority to ease out the overall process. So, now by default, both the original publisher and the organization owner will have admin privileges on that package. Checkout [this](https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages#visibility-and-access-permissions-for-container-images) document to know more. ☮️

**Comment by nyi851** [comment]
All is well

**Comment by 3kh0** [comment]
Amazing! Thanks for the update!

**Comment by ghost** [comment]
Thanks for the update!

**Comment by airtonix** [comment]
This needs to be resolved, it's a blocker for us. 

https://github.com/orgs/community/discussions/37570

Someone in our Org enabled this beta and we can't opt out now. I can't be sure but i believe it's causing some of our action runs to never end. (it started happening exactly when we were accepted into the large runner beta)

**Comment by jpadams** [comment]
@Akash1134 could you tell me the rate limits for public packages (container images) with Github Packages?
The docs suggest that storage and egress/data transfer are free for public packages, but I figure there must be a rate limit or something. Thank you!

related: https://github.com/orgs/community/discussions/27387#discussion-4256987

**Reply by Akash1134** [reply]
Hi @jpadams Please checkout this [doc](https://docs.github.com/en/billing/managing-billing-for-github-packages/managing-your-spending-limit-for-github-packages#managing-usage-and-spending-limit-email-notifications) for details.

**Comment by mikeee** [comment]
Are there any plans to enable runners to be shared account-wide rather than having to set up or migrate to orgs?
